### PR TITLE
[SRVCOM-2463, SRVCOM-2486, SRVCOM-2462, SRVKE-1490] Serverless release notes 1.30

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
`serverless-docs-1.30`+

Issue:
https://issues.redhat.com/browse/SRVCOM-2463
https://issues.redhat.com/browse/SRVCOM-2486
https://issues.redhat.com/browse/SRVCOM-2462
https://issues.redhat.com/browse/SRVKE-1490

Link to docs preview:
* GA/TP features table: https://64019--docspreview.netlify.app/openshift-serverless/latest/about/serverless-release-notes#serverless-tech-preview-features_serverless-release-notes
* Deprecated/removed features table: https://64019--docspreview.netlify.app/openshift-serverless/latest/about/serverless-release-notes#serverless-deprecated-removed-features_serverless-release-notes
* Serverless 1.30 release notes: https://64019--docspreview.netlify.app/openshift-serverless/latest/about/serverless-release-notes#serverless-rn-1-30-0_serverless-release-notes

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/serverless-deprecated-removed-features.adoc [hunk 1]
```diff
@@ -13,14 +13,39 @@ For the most recent list of major functionality deprecated and removed within {S
...
+|-
+|-
+|Deprecated
+|Deprecated
+|Deprecated
+
+|Serving and Eventing `v1alpha1` API
+|-
+|-
+|Deprecated
```
Comment (nak3) [inline modules/serverless-deprecated-removed-features.adoc:47]: Interesting... serverless 1.29,1.30 has not removed `v1alpha1` on Red Hat OpenShift Service on AWS even though  it was removed on OpenShift Container Platform and OpenShift Dedicated?
It is not a mistake?

**Review by nak3 — COMMENTED** [review]
LGTM for serving part except for one comment.

File: modules/serverless-deprecated-removed-features.adoc (hunk 1 above)
Comment (rh-max) [reply modules/serverless-deprecated-removed-features.adoc:47]: @nak3 Thanks Kenjiro. It was just a broken table, now the status for all OpenShift variants is the same.

File: modules/serverless-rn-1-30-0.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+[id="new-features-1-30-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.9.
+* {ServerlessProductName} now uses Knative Eventing 1.9.
+* {ServerlessProductName} now uses Kourier 1.9.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.9.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.9.
+* The `kn func` CLI plug-in now uses `func` FIXME.
```
Comment (ReToCode) [inline modules/serverless-rn-1-30-0.adoc:24]: Pretty sure you are aware of this, just not to forget :)

File: modules/serverless-rn-1-30-0.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+* Beginning {ServerlessProductName}, Kourier controller and gateway manifests have the following limits and requests by default:
+** requests:
+*** cpu: 200m
+*** memory: 200Mi
+** limits:
+*** cpu: 500m
+*** memory: 500Mi
++
+See the "Overriding Knative Serving system deployment configurations" section of {ServerlessProductName} documentation.
```
Comment (ReToCode) [inline modules/serverless-rn-1-30-0.adoc:54]: Can this be a link?

File: modules/serverless-rn-1-30-0.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+* The `NamespacedKafka` annotation, which was a Technology Preview (TP) feature, is now deprecated in favor of the standard Kafka broker with no data plane isolation.
+
+[id="fixed-issues-1-30_{context}"]
+== Fixed issues
+
+* Previously, the `3scale-kourier-gateway` pod was sending thousands of `net-kourier-controller` DNS queries daily. New queries were being sent for each `NXDOMAIN` reply. This continued until the correct DNS query was produced.
++
+The query now has the `net-kourier-controller.knative-serving-ingress` fully-qualified domain name (FQDN) by default, which solves the problem.
```
[the remark this answers is not in the document]
Comment (nak3) [reply modules/serverless-rn-1-30-0.adoc:63]: Ah, thank you. The final version became `net-kourier-controller.knative-serving-ingress.<cluster domain>.` 
(i.e. `net-kourier-controller.knative-serving-ingress.svc.cluster.local.` with cluster local domain + dot at the end.)

File: modules/serverless-rn-1-30-0.adoc (hunk 2 above)
Comment (rh-max) [reply modules/serverless-rn-1-30-0.adoc:24]: @ReToCode I am, but I never know the confirmed version :) Usually Lance or Jeff tells me.

File: modules/serverless-rn-1-30-0.adoc (hunk 3 above)
Comment (rh-max) [reply modules/serverless-rn-1-30-0.adoc:54]: @ReToCode Unfortunately, it cannot, as we cannot link from a module. However, I added this into the "Additional resources" heading after the 1.30 release notes section. Thanks!

File: modules/serverless-rn-1-30-0.adoc (hunk 2 above)
Comment (ReToCode) [reply modules/serverless-rn-1-30-0.adoc:24]: /cc @lkingland

Comment (rh-max) [reply modules/serverless-rn-1-30-0.adoc:24]: @jrangelramos Jeff, could you tell me the `kn func` version please?

File: modules/serverless-rn-1-30-0.adoc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+* {ServerlessProductName} now uses Kourier 1.9.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.9.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.9.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+
+* {FunctionsProductName} nows supports {pipelines-title} versions 1.10 and 1.11. Older versions of {pipelines-title} are no longer compatible with {FunctionsProductName}.
+
+* Eventing integration with {SMProductName} is now available as a Technology Preview (TP) feature.
++
+The integration includes the following:
```
Comment (mgencur) [inline modules/serverless-rn-1-30-0.adoc:30]: This list has a strange formatting, see picture
image

File: modules/serverless-rn-1-30-0.adoc [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+* {ServerlessProductName} now runs on HyperShift-hosted clusters.
+
+* {ServerlessProductName} now runs on single-node OpenShift.
+
+* {FunctionsProductName} nows supports {pipelines-title} versions 1.10 and 1.11. Older versions of {pipelines-title} are no longer compatible with {FunctionsProductName}.
+
+* Eventing integration with {SMProductName} is now available as a Technology Preview (TP) feature.
++
+The integration includes the following:
```
Comment (skopacz1) [inline modules/serverless-rn-1-30-0.adoc:34]: Suggestion: is there any noun that can be included here to be more specific? Such as "the integration includes the following resources" or "objects" or "features". No big deal if there's no one word to cover these list items.

File: modules/serverless-rn-1-30-0.adoc [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+* Pipelines-as-code for {FunctionsProductName} is now available as a Technology Preview (TP).
+
+* You can now configure the burst and queries per second (QPS) values for `net-kourier`.
+
+* {FunctionsProductName} users now have the ability to override the `readiness` and `liveness` probe values in the `func.yaml` file for individual Quarkus functions.
++
+See "Functions development reference guide" for guidance on Quarkus, TypeScript, and Node.js functions.
+
+* Beginning {ServerlessProductName}, Kourier controller and gateway manifests have the following limits and requests by default:
```
Comment (skopacz1) [inline modules/serverless-rn-1-30-0.adoc:55]: Should this be "Beginning with {ServerlessProductName} 1.3.0"?

File: modules/serverless-rn-1-30-0.adoc [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+* Previously, the `3scale-kourier-gateway` pod was sending thousands of `net-kourier-controller` DNS queries daily. New queries were being sent for each `NXDOMAIN` reply. This continued until the correct DNS query was produced.
++
+The query now has the `net-kourier-controller.knative-serving-ingress.svc.<cluster domain>.` fully-qualified domain name (FQDN) by default, which solves the problem.
+
+[id="known-issues-1-30-0_{context}"]
+== Known issues
+
+* Building and deploying a function using Podman version 4.6 fails with the `invalid pull policy "1"` error.
++
+To workaround this issue, use Podman version 4.5.
```
Comment (skopacz1) [inline modules/serverless-rn-1-30-0.adoc:79]: If using workaround as a verb, should it be two words?

workaround -> work around

**Review by skopacz1 — COMMENTED** [review]
Left a few comments, but otherwise looks good!

File: modules/serverless-rn-1-30-0.adoc (hunk 6 above)
Comment (rh-max) [reply modules/serverless-rn-1-30-0.adoc:34]: "Components" is the only option, but this would clash with another meaning of "components" on OpenShift.

**Comment by briandooley** [comment]
@nainaz Can you give the final approval for this?

**Pull request merged**